features/step_definitions/custom_web_steps.rb

```diff
@@ -92,14 +92,22 @@ def matches_path?(url, path)
 end
 
 Then /^I should see "([^"]*)" in the "([^"]*)" column and "([^"]*)" row$/ do |text, column, row|
-  row_element = find(:xpath, "//td/a[text()=\"#{row}\"]/ancestor::tr")
-  column_element = find(:xpath, "//th[text()='#{column}']")
+  if has_css?('table.pf-c-table')
+    column_values = find_all("td[data-label='#{column}'").map(&:text)
+    rows = find_all('tbody tr td[data-label="Group/Org."]')
+    index = rows.find_index{ |r| r.text == row }
 
-  row_index = row_element.path.match(/^.*(\d)\]$/)[1]&.to_i
-  column_index = column_element.path.match(/^.*(\d)\]$/)[1]&.to_i
+    actual_text = column_values[index]
+  else
+    # DEPRECTED!!!
```

Comment:
Typo :eyes:
(might come back to bite us in case we search by "deprecated" keyword later)